addons/website/static/src/components/fullscreen_indication/fullscreen_indication.js

```diff
@@ -25,7 +25,7 @@ export class FullscreenIndication extends Component {
     }
 
     get fullScreenIndicationText() {
-        return markup(sprintf(escape(_t("Press %(key)s to exit full string")), "<span>esc</span>"));
+        return markup(sprintf(escape(_t("Press %(key)s to exit full screen")), "<span>esc</span>"));
```

Comment:
@odoo/rd-security this is just a wording fix, inside a safe markup
The typo was introduced a few day ago